packages/core/package.json

```diff
@@ -27,6 +27,7 @@
     "directory": "packages/core"
   },
   "scripts": {
+    "audit-moderate": "npm install && npm audit --audit-level=moderate",
```

Comment:
We don't want other sibling packages in package-lock.json files since lerna manages but npm audit will fail saying it's not there - run npm install to fix.  

I also tried the other way of just checking in the lock file changes from npm install with the hash and the url from npm but then there's issues dev side if you change a package and want to consume those changes from a sibling package before publishing it.

This seems to work although not ideal.  I'm open to other suggestions.  Not a lerna expert ...  However, I do see there's an open issue in lerna to add a lerna audit first class command so maybe that's the ideal solution in the future.  https://github.com/lerna/lerna/issues/1663